code/game/machinery/doors/airlock.dm

```diff
@@ -1651,6 +1666,14 @@
 	emergency = !emergency
 	update_appearance()
 
+//SKYRAT EDIT ADDITION START
+/obj/machinery/door/airlock/proc/toggle_engineering(mob/user)
+	if(!user_allowed(user))
+		return
+	engineering_override = !engineering_override
```

Comment:
= FALSE for readability

Reply:
They are inverting a value. This is perfectly valid.

Reply:
yea, this is standard for toggling.

Reply:
This would make it so that new doors made afterwards would become out of sync with the others when the alert ends, so the new doors would OPEN.